BrightCoders-Proyectos/control-almacen
# Feature/6 sidebar

# Description
Implemented Sidebar menu 
-Made responsive.
-Added a simple design. It could be redesign if necessary
-Buttons added to work on the future features on the proyect. It could be added more buttons if needed

To test it:
- Clone the repository
-Then change to this branch with:
```
git checkout feature/6-sidebar
```
-start a local server with:
```
bundle exec rails s
```
-On your web browser go to: localhost:3000
-Add a new user
-Then you will be redirect to the index. On it you will see the sidebar on the left of the page and it will be responsive if you want to try it.
### Index page desktop screen
![Captura de pantalla de 2020-10-29 16-57-40](https://user-images.githubusercontent.com/51497719/97641327-43885200-1a08-11eb-8421-d842a4e78241.png)
### Index page small screen
![Captura de pantalla de 2020-10-29 16-58-01](https://user-images.githubusercontent.com/51497719/97641328-4420e880-1a08-11eb-8ccf-24ea487c41dc.png)
### Index page small screen, button active
![Captura de pantalla de 2020-10-29 16-58-03](https://user-images.githubusercontent.com/51497719/97641330-44b97f00-1a08-11eb-9701-da2f3f3d1bbe.png)
